platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/dsl/dependencies/DefaultDependencyHandler.java

```diff
@@ -244,6 +245,13 @@ public Dependency project(Map<String, ?> notation) {
     @Override
     @SuppressWarnings("rawtypes")
     public Dependency module(Object notation, @Nullable Closure configureClosure) {
+
+        DeprecationLogger.deprecate("Declaring client module dependencies")
```

Comment:
Can a more specific deprecation be used here, such as `deprecateAction`?

Reply:
I changed it to `deprecateAction` but that did not affect the deprecation message at all. Is there something wrong with the `.deprecate()` message that is printed already?

Reply:
It formats the advice if you use `replaceWith`. Updated to use `deprecateAction`